pull_into_place/pipeline.py

```diff
@@ -120,6 +120,14 @@ def rosetta_scripts_path(self):
     def rosetta_database_path(self):
         return self.rosetta_subpath('database')
 
+    @property
+    def rosetta_vall_path_small(self):
+        return self.rosetta_database_path + '/sampling/small.vall.gz'
```

Comment:
Instead of having both `rosetta_vall_path` and `rosetta_vall_path_small`, it might be better to just have `rosetta_vall_path(small=False)`.  That would also shorten the ternary expressions you have in the big job scripts and keep more of the file path logic in `Workspace` and its subclasses.

Reply:
Also, use `os.path.join()` instead of string concatenation.